wiki/Bancho_(server)/pt-br.md

```diff
@@ -0,0 +1,26 @@
+---
+tags:
+  - infrastructure
+---
+
+# Bancho (servidor)
+
+*Para outros usos, veja [Bancho (desambiguação)](/wiki/Disambiguation/Bancho).*
+
+**Bancho** (às vezes *osu!Bancho*) é um codinome do componente do servidor responsável por controlar em tempo real as interações entre jogadores utilizando as versões estáveis de osu!.
+
+- Sistema de Chat, [Presença Online](/wiki/Client/Interface/Chat_console#extended-chat-console), e o [Gateway de IRC](/wiki/Community/Internet_Relay_Chat)
+- [Modo Espectador](/wiki/Gameplay/Spectating)
+- [Sistema Multijogador](/wiki/Client/Interface/Multiplayer) e [Gerenciamento de Torneios](/wiki/osu!_tournament_client/osu!tourney/Tournament_management_commands)
+
+::: Infobox
+![](img/connection-warning.png "Screenshot of the main menu with the bad connection symbol (bottom right, first)")
```

Comment:
please translate the image caption as well

Reply:
Oops, didn't noticed it had a image caption. Fixed it.